go-fuzz-build/main.go

```diff
@@ -84,6 +85,18 @@ func main() {
 	// See also https://golang.org/issue/29824.
 	lits := c.gatherLiterals()
 	var blocks, sonar []CoverBlock
+
+	if *flagLibFuzzer == true {
+		archive := c.buildInstrumentedBinary(&blocks, nil)
+
+		if *flagOut == "" {
```

Comment:
How about we unify the decision about what to use for *flagOut. So above:

```go
if *flagOut == "" {
  suffix := ".zip"
  if *flagLibFuzzer {
    suffix = ".a"
  }
  *flagOut = c.pkgs[0].Name+"-fuzz"+suffix
}
```

Reply:
Then if/when we add the Fuzz method name as context, we won't have to add it in three places. (We already need to add it in go-fuzz-build and go-fuzz.)

Reply:
Since libFuzzer doesn't use the literals, let's move this whole thing up higher above the giant comment and make it standalone. Something like:

```
if *flagLibFuzzer {
  var blocks []CoverBlock
  archive := ...
  os.Rename ...
  return
}
```

Reply:
Does the literal finder extract hardcoded strings from the source files?
That might actually be very useful to create a dictionary file from, that libFuzzer can consume via the ```-dict=``` parameter. But I'll have to test this.

Reply:
Yes, it does.

Reply:
Should we also do the last step and run:
```
clang target.a -fsanitize=fuzzer
``
? This gives me a working fuzzer which is nice.

Reply:
This is interesting but won't work with oss-fuzz which requires that you link against ```-lFuzzingEngine```.

Reply:
OK, let's leave it as is for now.
We can get back to this if/when somebody will have interest in using it. We could build both, or have an additional flag.